src/Tools/VMC/Key.h

```diff
@@ -0,0 +1,33 @@
+#ifndef _VMC_KEY_H_
+#define _VMC_KEY_H_
+
+#include <ostream>
+
+namespace genie{
+namespace vmc{
+
+struct Key
+{
+  Key(int _nucl_pdg, int _nu_pdg, bool _iscc)
+    : nucl_pdg(_nucl_pdg), nu_pdg(_nu_pdg), iscc(_iscc) {}
```

Comment:
I'm sure we will cover this at the review today, but why only CC and NC? We know this is not enough.